go.mod

```
@@ -1,6 +1,6 @@
 module github.com/meitner-se/oto
 
-go 1.22
+go 1.22.0
```

Need to revert this, pipeline seems to fail due to it 
```suggestion
go 1.22
```

---

I'll test, something was failing before and this fixed it, but i'll test it again